CustomScript/customscript.py

```diff
@@ -405,6 +417,8 @@ def to_process(file_path, extensions=['.sh', ".py"]):
     for extension in extensions:
         if file_path.endswith(extension):
             return True
+        else:
+            return False
```

Comment:
this now makes anything below this line (and the rest of the elements in the `extensions` array) dead code.
 Add a line note